networking/openshift_sdn/migrate-to-openshift-sdn.adoc

```diff
@@ -8,6 +8,8 @@ toc::[]
 
 As a cluster administrator, you can migrate to the OpenShift SDN network plugin from the OVN-Kubernetes network plugin.
```

Comment:
This file does not render in the preview. Not sure why.

Reply:
@dfitzmau Thank you! I will re-check these two files.